### PR TITLE
A few fixes for Firestore bundle sdk builds

### PR DESCRIPTION
**Pull request by wu-hui**
* Add missing `firebase/firestore/bundle` and `firebase/firestore/memory/bundle`.
* Add bundle to firebase complete build
* Add missing firestore.proto for Firestore memory build for Node.

### CODE REVIEW

File: packages/firebase/rollup.config.js [hunk 1]
```diff
@@ -111,7 +117,7 @@ const appBuilds = [
 
 const componentBuilds = pkg.components
   // The "app" component is treated differently because it doesn't depend on itself.
-  .filter(component => component !== 'app')
+  .filter(component => component !== 'app' && component !== 'firestore')
```
Comment (schmidt-sebastian) [inline packages/firebase/rollup.config.js:120]: Please update comment.

File: packages/firebase/rollup.config.js [hunk 2]
```diff
@@ -149,17 +155,67 @@ const componentBuilds = pkg.components
...
+        format: 'cjs',
+        sourcemap: true
+      },
+      {
+        file: resolve('firestore', firestorePkg.module),
+        format: 'es',
+        sourcemap: true
+      }
+    ],
+    plugins,
```
Comment (schmidt-sebastian) [inline packages/firebase/rollup.config.js:173]: Do we not need

```
uglify({
           output: {
             ascii_only: true // escape unicode chars
            }	               
})
```

File: packages/firebase/src/index.rn.ts [hunk 3]
```diff
@@ -24,6 +24,7 @@ import '../database';
 //  `atob`). We should provide a RN build that works out of the box.
 import '../storage';
 import '../firestore';
+import '../firestore/bundle';
```
Comment (schmidt-sebastian) [inline packages/firebase/src/index.rn.ts:27]: The React Nativce build of the main Firestore package splits up bundles and non-bundled code. I think we should do the same here.

File: packages/firebase/rollup.config.js (hunk 1 above)
Comment (wu-hui) [reply packages/firebase/rollup.config.js:122]: Done.

File: packages/firebase/rollup.config.js (hunk 2 above)
Comment (wu-hui) [reply packages/firebase/rollup.config.js:175]: The index.ts is just `import '@firebase/firestore';` (so is the output), I think we can skip it here.

File: packages/firebase/src/index.rn.ts (hunk 3 above)
Comment (wu-hui) [reply packages/firebase/src/index.rn.ts:27]: This is for "complete" build, which include everything (except memory only builds) on other platforms.

**Review by schmidt-sebastian — APPROVED** [review]

**Pull request merged**